Royaleagle73/A1_Q1.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+void replace_cube(int a[],int n)
+{
+int i,count=1;
+for(i=0;i<5;i++)
+{
+if(i+1==n)
+{
+a[i]=(a[i]*a[i]*a[i]);  
+count=count+1;
+n=n*count;
+}
+}
+}
+int main()
+{ 
+int a[5]={1,2,3,4,5},n=2,i;
+replace_cube(a,n);
+for(i=0;i<5;i++)
+{
+printf("%d\t",a[i]);
+}    
+return 0;
```

Comment:
Can you please fix formatting of code for better readability. Also, add a new line at the end of file.

Reply:
![output](https://user-images.githubusercontent.com/110844185/195108136-ae83069c-9395-43ff-bb42-39ba7f98d2f3.png)

Reply:
Kindly check the new commit i made on the PR,thank you.

Reply:
under "Files changed" my updated code is avaliable